test/src/audits/manifest/short-name.js

```diff
@@ -34,15 +34,26 @@ describe('Manifest: short_name audit', () => {
 
   // Need to disable camelcase check for dealing with short_name.
   /* eslint-disable camelcase */
-  it('fails when a manifest contains no short_name', () => {
+  it('fails when a manifest contains no short_name and no name', () => {
```

Comment:
this test isn't doing what it says it's doing. Would you mind fixing it as one last thing while you're in here? :)

This is testing the actual manifest format, not the parsed manifest object. It just fails the `if (manifest)` test (since `manifest` there is actually `manifest.value`, which is undefined on the `inputs` object below) and so returns `hasShortName ` as false immediately instead of checking the values of `name` and `short_name` as it should be.

This test needs to be like the others, parsing the manifest first, then passing that in:
```
const inputs = JSON.stringify({
  name: null,
  short_name: null
});
const manifest = manifestParser(inputs);

return assert.equal(Audit.audit({manifest}).value, false);
```